Demo/AuthenticatorDemo/Info.plist

```diff
@@ -2,6 +2,17 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>com.googleusercontent.apps.108380595987-ujhrhknecrqli756i72gkcs4aaia6nhb</string>
```

Comment:
I recall a discussion that might have been similar to what I'm about to write, but it might have been in an internal/private Slack channel so I'm taking the occasion for having it in a searchable medium.

Upon first looking at this, I though it was a security risk because it clearly exposes information relevant to the WordPress app only.

However:

- This is already committed [in the WordPress iOS repo](https://github.com/wordpress-mobile/WordPress-iOS/blob/1b266dbd810b2982b9ca37996d85660fa94490ab/WordPress/Info.plist#L490)
- Any savvy bad actor would be able to extract it from the app at runtime

I'm open to the idea of injecting this at build time in a more secure way like we do for `APICredentials` but, because this value has already "leaked", I would do it in a set of dedicated PR across all our apps.

Reply:
I don't think it's a risk to expose the client id.

> The client identifier is not a secret;

According to [this OAuth 2 speficication](https://www.rfc-editor.org/rfc/rfc6749#section-2.2).

Reply:
Even better 😄